config/default_records/plate_purposes/013_lcm_triomics_plate_purposes.wip.yml

```diff
@@ -0,0 +1,6 @@
+# Plate purposes for LCM Triomics DNA WGS and EMSeq
```

Comment:
Can we drop the word DNA from this comment? 

The Automation of WGS/EM-Seq - Pipeline Overview document lists the pipelines. Based on the listing, the following simplifies the naming:

DNA only -> WGS
DNA & Methylation -> **WGS and EMSeq**
RNA only -> RNASeq
DNA & RNA -> WGS and RNASeq
DNA, RNA & Methylation -> WGS, RNASeq and EMSeq

Note that I avoided using dash in the names, EMSeq and RNASeq to make it simpler, config keys, variable names, and visible names will look the same as single tokens.